lib/Net/Async/Blockchain/Client/ZMQ.pm

```diff
@@ -203,7 +203,7 @@ sub subscribe {
         my $handle = IO::Async::Handle->new(
             read_handle => $io,
             on_closed   => sub {
-                eval { close($io) };
+                eval { close($io) } // undef;
```

Comment:
Do we still need the `eval BLOCK` here?  `close()` itself doesn't throw an exception so I think the eval doesn't really catch anything.